src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -59,6 +59,29 @@ public abstract class SslStreamStreamToStreamTest
             }
         }
 
+        [Fact]
+        public async Task SslStream_ServerLocalCertificateSelectionCallbackReturnsNull_Throw()
```

Comment:
This test is formatted differently from the others. I did this to keep the format matching the rest of the tests in the file.